### PR TITLE
Fix: Padding required in the Facebook Messenger option in Livechat

### LINKED ISSUE #16041: Padding required in the Facebook Messenger option in Livechat
### Steps to reproduce:

1. Livechat settings as administrator.
2. Go to facebook messenger now

### Expected behavior:

Padding should be there
### Actual behavior:

Selection_001

### Server Setup Information:

- Version of Rocket.Chat Server: 3.0.0-develop
- Operating System: ubuntu

### PR DESCRIPTION
**Pull request by ritwizsinha**
Closes #16041 
Selection_011
In ref to #16042

### CODE REVIEW

**Comment by ritwizsinha** [comment]
@gabriellsh please review

File: app/livechat/client/views/app/integrations/livechatIntegrationFacebook.html [hunk 1]
```diff
@@ -9,8 +9,10 @@
 					{{else}}
 						<button class="button primary enable" {{enableButtonDisabled}}>{{_ "Enable"}}</button>
 						{{#unless hasToken}}
+							<div class="rc-input__wrapper">
 							<p>{{_ "You_have_to_set_an_API_token_first_in_order_to_use_the_integration"}}</p>
```
Comment (mariaeduardacunha) [inline app/livechat/client/views/app/integrations/livechatIntegrationFacebook.html]: Please, there are some missing tabs inside this div. Could you fix?

**Comment by mariaeduardacunha** [comment]
Next time please, commit your changes to the same Pull Request that you initially open. (#16042)

File: app/livechat/client/views/app/integrations/livechatIntegrationFacebook.html (hunk 1 above)
Comment (ritwizsinha) [reply app/livechat/client/views/app/integrations/livechatIntegrationFacebook.html]: @mariaeduardacunha I have made the changes please review

**Comment by ritwizsinha** [comment]
> Next time please, commit your changes to the same Pull Request that you initially open. (#16042)

Sure, actually I had to reinstall my os which led to this

**Pull request merged**